### PR TITLE
EWPP-873: Text with featured media paragraph type V2

### PR DESCRIPTION
**Pull request by yenyasinn**

### CODE REVIEW

File: modules/oe_paragraphs_media/oe_paragraphs_media.post_update.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  foreach ($field_config as $config) {
+    $config_record = $storage->read($config);
+    $entity_type = $config_manager->getEntityTypeIdByName($config);
+    $entity_storage = $entity_type_manager->getStorage($entity_type);
+    $entity = $entity_storage->load($config_record['id']);
+    if ($entity) {
+      // Skip if configuration exists.
+      continue;
+    }
+    $entity = $entity_storage->createFromStorageRecord($config_record);
```
Comment (ademarco) [inline modules/oe_paragraphs_media/oe_paragraphs_media.post_update.php:54]: We need to add config hash here, or strings contained in these configurations won't be translatable.

File: modules/oe_paragraphs_media/config/install/field.field.paragraph.oe_text_feature_media.field_oe_feature_media_title.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    - field.storage.paragraph.field_oe_feature_media_title
+    - paragraphs.paragraphs_type.oe_text_feature_media
+id: paragraph.oe_text_feature_media.field_oe_feature_media_title
+field_name: field_oe_feature_media_title
+entity_type: paragraph
+bundle: oe_text_feature_media
+label: Title
+description: ''
+required: false
+translatable: true
```
Comment (ademarco) [inline modules/oe_paragraphs_media/config/install/field.field.paragraph.oe_text_feature_media.field_oe_feature_media_title.yml:14]:
```suggestion
translatable: false
```

File: modules/oe_paragraphs_media/config/install/field.field.paragraph.oe_text_feature_media.field_oe_link.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  module:
+    - link
+id: paragraph.oe_text_feature_media.field_oe_link
+field_name: field_oe_link
+entity_type: paragraph
+bundle: oe_text_feature_media
+label: Link
+description: ''
+required: false
+translatable: true
```
Comment (ademarco) [inline modules/oe_paragraphs_media/config/install/field.field.paragraph.oe_text_feature_media.field_oe_link.yml:16]:
```suggestion
translatable: false
```

File: modules/oe_paragraphs_media/config/post_updates/00001_add_link_title_fields/field.field.paragraph.oe_text_feature_media.field_oe_feature_media_title.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    - field.storage.paragraph.field_oe_feature_media_title
+    - paragraphs.paragraphs_type.oe_text_feature_media
+id: paragraph.oe_text_feature_media.field_oe_feature_media_title
+field_name: field_oe_feature_media_title
+entity_type: paragraph
+bundle: oe_text_feature_media
+label: Title
+description: ''
+required: false
+translatable: true
```
Comment (ademarco) [inline modules/oe_paragraphs_media/config/post_updates/00001_add_link_title_fields/field.field.paragraph.oe_text_feature_media.field_oe_feature_media_title.yml:14]:
```suggestion
translatable: false
```

File: modules/oe_paragraphs_media/config/post_updates/00001_add_link_title_fields/field.field.paragraph.oe_text_feature_media.field_oe_link.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  module:
+    - link
+id: paragraph.oe_text_feature_media.field_oe_link
+field_name: field_oe_link
+entity_type: paragraph
+bundle: oe_text_feature_media
+label: Link
+description: ''
+required: false
+translatable: true
```
Comment (ademarco) [inline modules/oe_paragraphs_media/config/post_updates/00001_add_link_title_fields/field.field.paragraph.oe_text_feature_media.field_oe_link.yml:16]:
```suggestion
translatable: false
```

File: modules/oe_paragraphs_media/config/install/field.field.paragraph.oe_text_feature_media.field_oe_title.yml [hunk 6]
```diff
@@ -8,7 +8,7 @@ id: paragraph.oe_text_feature_media.field_oe_title
 field_name: field_oe_title
 entity_type: paragraph
 bundle: oe_text_feature_media
-label: Title
+label: Heading
 description: ''
 required: false
 translatable: true
```
Comment (ademarco) [inline modules/oe_paragraphs_media/config/install/field.field.paragraph.oe_text_feature_media.field_oe_title.yml:14]:
```suggestion
translatable: false
```

**Review by ademarco — CHANGES_REQUESTED** [review]

File: modules/oe_paragraphs_media/oe_paragraphs_media.post_update.php (hunk 1 above)
Comment (yenyasinn) [reply modules/oe_paragraphs_media/oe_paragraphs_media.post_update.php:56]: Added

**Review by sergepavle — APPROVED** [review]

**Review by ademarco — APPROVED** [review]

**Pull request merged**